src/privatesend-client.cpp

```diff
@@ -82,7 +82,7 @@ void CPrivateSendClientManager::ProcessMessage(CNode* pfrom, const std::string&
                 }
             }
         } else {
-            LOCK(cs_vecsessions); // have to lock this first to avoid deadlocks with cs_vecqueue
+            LOCK(cs_deqsessions); // have to lock this first to avoid deadlocks with cs_vecqueue
```

Comment:
nit: `cs_vecqueue` still in comment

Reply:
Hmmm... and...? vecqueue != vecsession :)

Reply:
You got me there. 😳